torch/_inductor/fx_passes/quantization.py

```diff
@@ -2529,6 +2529,190 @@ def _register_qlinear_weight_prepack():
         )
 
 
+def _register_smooth_quant_int_mm_pattern():
+    """
+    The pattern is:
+      (no bias) reshape -> _int_mm -> convert_element_type -> (expand -> mul) -> mul -> reshape
+    or
+      (with bias) pattern_no_bias -> add -> reshape -> reshape
+    """
+    pattern_no_bias = CallFunction(
```

Comment:
wondering if https://github.com/pytorch/pytorch/pull/139102 will be able to help simplify the pattern as well

Reply:
Thanks for the info. Can `InvokeQuant` represent the pattern of quantization or any pattern?

Reply:
it's mostly targeting dequant I think, see: https://github.com/pytorch/pytorch/pull/139102#discussion_r1821576857

Reply:
I see. Do you suggest waiting for that PR landing first then using `InvokeQuant` in this PR?

Reply:
Got it. Thanks.